tests/mtgjson5/providers/README_CASSETTES.md

```diff
@@ -0,0 +1,80 @@
+# VCR Cassettes for TCGplayer Tests
```

Comment:
i wonder if the AI did this or vcr bootstrapped it. it should go into docs/tests/cassettes.md